modules/migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/SchemaMigrator.java

```diff
@@ -65,7 +65,7 @@ public void migrate() throws MigrationClientException {
             String databaseType = DatabaseCreator.getDatabaseType(this.conn);
             if ("mysql".equals(databaseType)) {
                 Utility.setMySQLDBName(conn);
-                if (this.conn.getMetaData().getDatabaseProductVersion().startsWith("5.7")) {
+                if (getDatabaseProductVersion() > Float.parseFloat("5.7")) {
```

Comment:
can't we use the number 5.7 instead of using Float.parseFloat() ?

Reply:
Fixed in 34a21322715f17b7a8c30523c7a6af1c15892f01.